src/components/forms/RadioField/RadioField.js

```diff
@@ -0,0 +1,96 @@
+import {
+  Fieldset,
+  FieldsetLegend,
+  FormField,
+  FormLabel,
+  Paragraph,
+  RadioButton,
+} from '@utrecht/component-library-react';
+import {Field, useFormikContext} from 'formik';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import {HelpText, ValidationErrors, Wrapper} from 'components/forms';
+import {LabelContent} from 'components/forms/Label';
+
+import './RadioField.scss';
+
+/**
+ * A radio field with a set of options.
+ *
+ * @reference https://nl-design-system.github.io/utrecht/storybook-react/?path=/docs/react-component-form-field-radio-group--docs
+ */
```

Comment:
What makes the Utrecht radio group not reusable? I'm thinking of an adaptor component that handles  our Formik state and the prop shape differences between the API of RadioField and the Utrecht Component?

Reply:
I'm not sure what you mean, this is following the example from the utrecht radio group :thinking:

Reply:
clarified - there's no magic component that I missed and Chris caught